modmesh/app/euler1d.py

```diff
@@ -545,14 +589,11 @@ def march_alpha2(self, steps=1):
             self.log(mm.time_registry.report())
         self.plt_mgr.update_lines()
 
-    def setup_solver(self, interval):
+    def setup_timer(self, interval):
```

Comment:
Since solver related flow has been removed from this function, only remaining timer related flow, therefore I changed the function name.

Reply:
Sounds good to me.